openfasoc/generators/glayout/glayout/flow/blocks/composite/cdac/mimcap_array.py

```diff
@@ -0,0 +1,717 @@
+from typing import List
+
+# Primitives
+from glayout.flow.primitives.mimcap import mimcap
+from glayout.flow.primitives.mimcap import mimcap_array
+from glayout.flow.primitives.via_gen import via_stack, via_array
+
+# Standard
+from glayout.flow.pdk.mappedpdk import MappedPDK
+from glayout.flow.pdk.sky130_mapped import sky130_mapped_pdk as sky130
+
+# gdsfactory
+from gdsfactory import Component
+from gdsfactory.cell import cell
+from gdsfactory.components.rectangle import rectangle
+
+# Utility
+from glayout.flow.pdk.util.comp_utils import evaluate_bbox, prec_center, prec_array, align_comp_to_port
+
+# Routing
+from glayout.flow.routing.straight_route import straight_route
+from glayout.flow.routing.c_route import c_route
+
+'''
+def dac_mimcap_array_netlist(pdk: MappedPDK) -> Netlist:
+    return Netlist(
+        circuit_name = "DAC_MIMCAP_ARR",
+
+        nodes = ['V1','V2','V3',],
+        source_netlist=,
+        instance_format=,
+        parameters={
+            'model': pdk.models['mimcap'],
+            'length': size[0]
+            'width': size[1]
+        }    
+    )
+'''
+
+def create_6bit_dac_mimcap_array(pdk: MappedPDK):
```

Comment:
Was the existing mimcap array generator inadequate? If yes, how? Perhaps this variant can be incorporated into the same function as an argument so it can be used elsewhere as well. It would also reduce code duplication.